src/drive/locales/en.json

```diff
@@ -87,6 +90,14 @@
         "action": "Authorize access",
         "success": "The application has access to your contacts"
       }
+    },
+    "banner": {
+      "shared_from": "Shared from the cozy of",
```

Comment:
I think it'd be better English to have `Shared from %{name}'s cozy`.

Reply:
783eaa6ef7705148c2eeb0b44e08aba34aeef3de